webapp/channels/src/components/suggestion/emoticon_provider.tsx

```diff
@@ -39,10 +39,9 @@ const EmoticonSuggestion = React.forwardRef<HTMLLIElement, SuggestionProps<Emoji
         >
             <div className='pull-left emoticon-suggestion__image-container'>
                 <img
-                    alt={text}
                     className='emoticon-suggestion__image'
                     src={getEmojiImageUrl(emoji)}
-                    title={text}
+                    alt=''
```

Comment:
For emoji suggestions, we don't need to explicitly add a label/description to them because the only text in them is the name of the emoji which is automatically used as the label.

I removed the title and alt text from the image to prevent the emoji's name from being read out twice